src/pages/BuyDomain/BuyDomain.tsx

```diff
@@ -0,0 +1,70 @@
+import type { FC } from 'react';
+
+import { useState } from 'react';
+import {
+	IconZero,
+	DomainSearch,
+	BuyDomainModal,
+	ConnectWallet,
+} from '../../components';
+import { useWeb3 } from '../../hooks';
+import styles from './BuyDomain.module.scss';
+
+export const BuyDomain: FC = () => {
+	const { account } = useWeb3();
+
+	const [buyDomainModal, setBuyDomainModal] = useState<{
+		isOpen;
+		domainName;
+	}>({
+		isOpen: false,
+		domainName: '',
+	});
+
+	const handleOnBuyButtonClick = (domainName) => {
+		setBuyDomainModal({
+			isOpen: true,
+			domainName,
+		});
+	};
+
+	const handleOnModalClose = () => {
```

Comment:
Remove `handle` from these names, change `onBuyButtonClick` to `onClickBuyButton`.

Reply:
Updated